.github/CONTRIBUTING.md

```diff
@@ -18,18 +18,18 @@ Hi! I'm really excited that you are interested in contributing to Vue.js. Before
 
 - Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
 
-- Work in the `src` folder and **DO NOT** checkin `dist` in the commits.
+- Work in the `src` folder and **DO NOT** check the `dist` in the commits.
 
 - It's OK to have multiple small commits as you work on the PR - GitHub will automatically squash it before merging.
 
-- Make sure `npm test` passes. (see [development setup](#development-setup))
+- Make sure the `npm test` passes. (see [development setup](#development-setup))
```

Comment:
No need for `the` here. It's not really helpful, as it's implied the reader understands `npm` scripts at this point.